requirements/specifications/metrics/media-metrics.md

```diff
@@ -0,0 +1,293 @@
+
+## 2. Table of Contents
+- [1. Overview](#1-overview)
+- [2. Table of Contents](#2-table-of-contents)
+- [3. Content Metrics](#3-content-metrics)
+  - [3.1. Content Start](#31-content-start)
+  - [3.2. Content Stop](#32-content-stop)
+- [4. Media Metrics](#4-media-metrics)
+  - [4.1. Load Start](#41-load-start)
+  - [4.2. Play](#42-play)
+  - [4.3. Playing](#43-playing)
+  - [4.4. Pause](#44-pause)
+  - [4.5. Waiting](#45-waiting)
+  - [4.6. Progress](#46-progress)
+  - [4.7. Seeking](#47-seeking)
+  - [4.8. Seeked](#48-seeked)
+  - [4.9. Rate Change](#49-rate-change)
+  - [4.10. Rendition Change](#410-rendition-change)
+  - [4.11. Ended](#411-ended)
+  - [4.12. Error](#412-error)
+
+## 3. Content Metrics
+The term "content" is used to denote *any* type of primary content that a user could be engaged in, e.g. a video game, a live broadcast, or a pre-recorded video presentation.
+
+### 3.1. Content Start
+The `Metrics` module **MUST** include a `startContent` API so that an app may report when the user is engaged in "primary content."
+
+The `startContent` API **MUST** have an optional `entityId` parameter for the app to specify which content is being started.
+
+The remainder of this section outlines guidelines that apps should consider when calling `startContent`.
+
+In order for content to be considered started, the user **SHOULD** have requested it through some explicit interaction with the app.
+
+If an app automatically starts some default content, then the act of launching the app to the foreground **MAY** be considered an explicit interaction to start the content.
+
+An app **SHOULD** differentiate, as appropriate, between previewing content currently in focus in a browse experience versus content the user is truly engaged with.
+
+Once content has been identified to be started, apps **SHOULD** wait until any loading is complete before calling `startContent`.
+
+If the content is streaming media content, and will leverage the Media Metrics APIs, then `startContent` **SHOULD** be called immediately before the first `mediaPlaying` call.
+
+If content requires user interaction before starting, e.g. a prompt to select language, region, game difficulty, etc, apps **SHOULD** wait until all of such prompts are completed before calling `startContent`.
+
+### 3.2. Content Stop
+The `Metrics` module **MUST** include an `stopContent` API so that an app may report when the user finished being engaged in "primary content."
+
+The `stopContent` API **MUST** have an optional `entityId` parameter for the app to specify which content is being stopped.
+
+The remainder of this section outlines guidelines that apps should consider when calling `stopContent`.
+
+In order for content to be considered stopped, it **MUST** have previously been started within the app's current lifecycle.
+
+Content **MAY** be stopped explicitly by the user, or automatically by the app.
```

Comment:
Do we have to differentiate between the end of the "user content" (what the user is primarily interested in) and the "media content" (the entire media file)?

For instance, once a show ends and the credits start to roll, many apps will show an "up next" experience to start the next episode.  Should we surface an API so apps can denote when such an experience starts?